src/main.ts

```diff
@@ -1,10 +1,34 @@
 // TASK: Get data from console into program
 
+// TASK: Send data to backend
+
 import * as readline from 'readline';
```

Comment:
I noticed that you're still using readline here, but you remove it as a dependency in your `package.json`... This will cause problems. On this branch in your terminal, type in `npx yarn add readline`